src/main.ts

```diff
@@ -630,14 +630,18 @@ class NixInstallerAction {
   }
 
   private async setup_kvm(): Promise<boolean> {
+    const current_user = userInfo();
+    const is_root = current_user.username === "root";
```

Comment:
Sometimes the root username can be known as `toor`. How about checking if the uid is 0?

```suggestion
    const is_root = current_user.uid === 0;
```